spread.yaml

```diff
@@ -31,6 +31,9 @@ backends:
       - ubuntu-22.04-64:
           workers: 4
           storage: 40G
+      - ubuntu-24.04-64:
```

Comment:
hmm do we need this?

Reply:
Oops! No, not right now. That was related to me trying some stuff out and forgetting to remove it.